src/lib/coingecko/utils.ts

```diff
@@ -0,0 +1,172 @@
+import { ASSET_NAMESPACE, bscChainId, toAssetId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+
+import {
+  COINGECKO_NATIVE_ASSET_ID_TO_ASSET_ID,
+  COINGECKO_PLATFORM_ID_TO_CHAIN_ID,
+} from './constants'
+import type {
+  CoingeckoAsset,
+  CoingeckoAssetDetails,
+  MoversResponse,
+  RecentlyAddedResponse,
+  TrendingResponse,
+} from './types'
+
+const coingeckoBaseUrl = 'https://api.proxy.shapeshift.com/api/v1/markets'
```

Comment:
Not in env but not in env either in markets slice, meh

Reply:
Already existing in the caip package, do we really need to duplicate it?

Reply:
Yeah just following the pattern there, don't feel like we need to add an env in this diff?